inhouse_bot/cogs/admin_cog.py

```diff
@@ -121,15 +134,18 @@ async def mark(self, ctx: commands.Context, channel_type: str):
         Marks the current channel as a queue or ranking channel
         """
         if channel_type.upper() == "QUEUE":
-            queue_channel_handler.mark_queue_channel(ctx.channel.id, ctx.guild.id)
-
-            await ctx.send(f"Current channel marked as a queue channel")
+            if queue_channel_handler.mark_queue_channel(ctx.channel.id, ctx.guild.id):
```

Comment:
I feel like you can use the helper functions supplied already

```
        if channel_type.upper() == "QUEUE":
            if queue_channel_handler.is_queue_channel(ctx.channel.id):
                   await ctx.send(f"Current channel is already a queue channel")
            else:
                   queue_channel_handler.mark_queue_channel(ctx.channel.id, ctx.guild.id)
```

I think its more clearer this way, the mark_queue_channel function doesnt return a boolean but only going to the else block because of like an exception or duplicate error or something

Reply:
Fixed